python/requirements.txt

```diff
@@ -2,7 +2,7 @@
 # This file is autogenerated by pip-compile with Python 3.12
 # by the following command:
 #
-#    ./bash/compile_python_packages.sh python
+#    pip-compile --output-file=python/requirements.txt python/requirements.in
```

Comment:
can u compile via `./bash/compile_python_packages.sh python`? especially because it deletes `requirements.txt` before compiling which is needed for the `.txt` to fully reflect the resolved `.in`

Reply:
Ah didn't realize that was default behavior. But if you run with `--upgrade` you don't need to delete requirements.txt

Reply:
realpath doesn't work properly on my mac, so the script doesn't run properly. Do you have no issues with it?

Reply:
Or at least the realpath command in the script doesn't work for me

Reply:
damn i forgot about macbook realpath pain